src/components/accordion/__tests__/CdrAccordion.spec.js

```diff
@@ -1,212 +1,158 @@
-
-    // opened state
-    button.trigger('click');
-    wrapper.setProps({ opened: true }); // fake the opening logic
-    await wrapper.vm.$nextTick();
-    expect(button.attributes('aria-expanded')).toBe('true');
-    expect(contentArea.attributes('aria-hidden')).toBe('false');
   })
 
-  describe('accordion data setup', () => {
-    it('sets maxHeight when starting closed', () => {
-      const wrapper = shallowMount(CdrAccordion, {
-        propsData: {
-          id: 'test',
-          level: '2',
-          opened: false,
-        },
-        slots: {
-          default: 'This is some slot text.',
-          label: 'label',
-        },
+  describe('when shallow mounted', () => {
+    let wrapper;
+    let button;
+    let contentArea;
+    beforeEach(() => {
+      wrapper = shallowMount(CdrAccordion, { ...baseComponentPattern });
+      button = wrapper.find('#test');
+      contentArea = wrapper.find('.cdr-accordion__content');
+    })
+    describe('when the accordion is closed', () => {
+      it('sets maxHeight to be 0', () => {
+        expect(wrapper.vm.maxHeight).toBe(0);
       });
 
-      expect(wrapper.vm.maxHeight).toBe(0);
-    });
-
-    it('maxHeight is adjusted in created when starting open', () => {
-      const wrapper = shallowMount(CdrAccordion, {
-        propsData: {
-          id: 'test',
-          level: '2',
-          opened: true,
-        },
-        slots: {
-          default: 'This is some slot text.',
-          label: 'label',
-        },
+      it('sets the "isOpenClass" computed prop to "cdr-accordion--closed" ', async () => {
+        expect(wrapper.vm.isOpenClass).toEqual('cdr-accordion--closed');
       });
-
-      expect(wrapper.vm.maxHeight).toBe('none');
-    });
-  });
-
-  describe('toggling accordion', () => {
-    it('updates emits an event onClick', async () => {
-      const wrapper = shallowMount(CdrAccordion, {
-        propsData: {
-          id: 'test',
-          level: '2',
-        },
-        slots: {
-          default: 'This is some slot text.',
-          label: 'label',
-        },
+      describe('when an accordion button receives focus', () => {
+        beforeEach(() => {
+          button.trigger('focus');
+        })
+        it('should be focused', ()=>{
+          expect(wrapper.vm.focused).toBeTruthy();
+        })
+        it('should have the proper focus class', ()=>{
+          expect(wrapper.classes()).toContain('cdr-accordion--focused');
+        })
+        describe('when an accordion button receives focus', () => {
+          beforeEach(() => {
+            button.trigger('blur');
+          });
+          it('should NOT be focused', ()=>{
+            expect(wrapper.vm.focused).toBeFalsy();
+          })
+          it('should NOT have the proper focus class', ()=>{
+            expect(wrapper.classes()).not.toContain('cdr-accordion--focused');
+          })
+        });
+      })
+      describe('a11y requirements', () => {
```

Comment:
There are two separate describes for "a11y requirements". I think it makes sense to have top level describes for the state of a component, and nested "a11y requirements" for each one